src/app.ts

```diff
@@ -1457,7 +1469,7 @@ export const initializeAppModules = () => {
       name: "closeStageView",
       initializedRequired: true,
       action: function () {
-        microsoftTeams.stageView.self.close();
+        (microsoftTeams.stageView as any).self.close();

```

Comment:
Is it breaking without this?

Reply:
I think that's because v2.25.0 of the teams-js doesn't yet have this API. @amyzhao1321, try updating the teams-js version in package.json to v2.26.0.

Reply:
Yes, the build succeeded after updating the Test app using @microsoft/teams-js: ^2.26.0. The yarn.lock file was updated after running the build. Please take a look to see if anything else needs updating, as the dependency version has changed.